### PR TITLE
Add unit tests (resolves #5)

### LINKED ISSUE #5: Add unit tests

### PR DESCRIPTION
**Pull request by jpfeil**
resolves #5

### CODE REVIEW

File: test/test_input/hook3_ret.results.tsv [hunk 1]
```diff
@@ -0,0 +1 @@
+HOOK3	ENSG00000168172	+	8	42823357	TAAGAAAGGCCAACGCAGCGCGAAGTCAACTTGAAACCTACAAGAGACAG	RET	ENSG00000165731	+	10	43612032	GAGGATCCAAAGTGGGAATTCCCTCGGAAGAACTTGGTTCTTGGAAAAAC
```
Comment (arkal) [inline test/test_input/hook3_ret.results.tsv]: Shouldn't this be in the fusion test?

File: test/test_input/test.pc_transcripts.fa [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
...ATGACTGGATTTGCCAACAAAGATAAGCAAGAACTTAAGCATGCTGGAAAGATAGCAACTGAAGCTTTGGAGAATATACGTACTATAGTGTCATTAACAAGGGAAAAAGCCTTCGAGCAAATGTATGAAGAGATGCTTCAGACTCAACACAGAAATACCTCGAAGAAAGCACAGATTATTGGAAGCTGTTATGCATTCAGCCATGCCTTTATATATTTTGCCTATGCGGCAGGGTTTCGATTTGGAGCCTATTTAATTCAAGCTGGACGAATGACCCCAGAGGGCATGTTCATAGTTTTTACTGCAATTGCATATGGAGCTATGGCCATCGGAGAAACGCTCGTTTTGGCTCCTGAATATTCCAAAGCCAAATCGGGGGCTGCGCATCTGTTTGCCTTGTTGGAAAAGAAACCAAATATAGACAGCCGCAGTCAAGAAGGGAAAAAGCCAGACACATGTGAAGGGAATTTAGAGTTTCGAGAAGTCTCTTTCTTCTATCCATGTCGCCCAGATGTTTTCATCCTCCGTGGCTTATCCCTCAGTATTGAGCGAGGAAAGACAGTAGCATTTGTGGGGAGCAGCGGCTGTGGGAAAAGCACTTCTGTTCAACTTCTGCAGAGACTTTATGACCCCGTGCAAGGACAAGTGCTGTTTGATGGTGTGGATGCAAAAGAATTGAATGTACAGTGGCTCCGTTCCCAAATAGCAATCGTTCCTCAAGAGCCTGTGCTCTTCAACTGCAGCATTGCTGAGAACATCGCCTATGGTGACAACAGCCGTGTGGTGCCATTAGATGAGATCAAAGAAGCCGCAAATGCAGCAAATATCCATTCTTTTATTGAAGGTCTCCCTGAGAAATACAACACACAAGTTGGACTGAAAGGAGCACAGCTTTCTGGCGGCCAGAAACAAAGACTAGCTATTGCAAGGGCTCTTCTCCAAAAACCCAAAATTTTATTGTTGGATGAGGCCACTTCAGCCCTCGATAATGACAGTGAGAAGGTGGTTCAGCATGCCCTTGATAAAGCCAGGACGGGAAGGACATGCCTAGTGGTCACTCACAGGCTCTCTGCAATTCAGAACGCAGATTTGATAGTGGTTCTGCACAATGGAAAGATAAAGGAACAAGGAACTCATCAAGAGCTCCTGAGAAATCGAGACATATATTTTAAGTTAGTGAATGCACAGTCAGTGCAGTGA
```
Comment (arkal) [inline test/test_input/test.pc_transcripts.fa]: also fusion test?

**Comment by arkal** [comment]
Looks good except for the fusion artifacts. Will merge once it is fixed.

**Comment by jpfeil** [comment]
@arkal

**Comment by jpfeil** [comment]
@arkal

**Comment by arkal** [comment]
Looking great!

**Pull request merged**